status/projects.py

```diff
@@ -575,7 +575,7 @@ def get(self, project, sample, step):
     @staticmethod
     def get_caliper_image(url):
         """returns a base64 string of the caliper image asked"""
-        #url pattern: sftp://genologics.scilifelab.se/home/glsftp/clarity/year/month/processid/artifact-id-file-id.png
+        #url pattern: sftp://ngi-lims-prod.scilifelab.se/home/glsftp/clarity/year/month/processid/artifact-id-file-id.png
```

Comment:
So the server name that we ssh to will be renamed too?

Reply:
Yes. genologics.scilifelab.se will be dismissed from service.